src/components/comment-list-item.component.js

```diff
@@ -13,71 +13,71 @@ import moment from 'moment/min/moment-with-locales.min';
 import { translate } from 'utils';
 import { colors, fonts, normalize } from 'config';
 
-const styles = StyleSheet.create({
-  container: {
-    paddingRight: 10,
-    paddingTop: 10,
-    backgroundColor: 'transparent',
-  },
-  header: {
-    flexDirection: 'row',
-    marginLeft: 10,
-    alignItems: 'center',
-  },
-  avatarContainer: {
-    backgroundColor: colors.greyLight,
-    borderRadius: 17,
-    width: 34,
-    height: 34,
-  },
-  avatar: {
-    width: 34,
-    height: 34,
-    borderRadius: 17,
-  },
-  titleSubtitleContainer: {
-    justifyContent: 'center',
-    flex: 1,
-    marginLeft: 10,
-  },
-  dateContainer: {
-    flex: 0.15,
-    alignItems: 'flex-end',
-    justifyContent: 'center',
-  },
-  linkDescription: {
-    ...fonts.fontPrimaryBold,
-    fontSize: normalize(13),
-    color: colors.primaryDark,
-  },
-  date: {
-    color: colors.greyDark,
-  },
-  commentContainer: {
-    paddingTop: 5,
-    marginLeft: 54,
-    borderBottomColor: colors.greyLight,
-    borderBottomWidth: 1,
-  },
-  commentBottomPadding: {
-    paddingBottom: 15,
-  },
-  commentText: {
-    fontSize: normalize(12),
-    color: colors.primaryDark,
-  },
-  commentTextNone: {
-    ...fonts.fontPrimary,
-    color: colors.primaryDark,
-    fontStyle: 'italic',
-  },
-  actionButtonIconContainer: {
-    paddingTop: 5,
-    paddingBottom: 10,
-    alignItems: 'flex-end',
-    justifyContent: 'center',
-  },
-});
+const Container = styled.View`
+  padding: 10px 10px 0 0;
+  background-color: transparent;
+`;
+
+const Header = styled.View`
+  flex-direction: row;
+  margin-left: 10px;
```

Comment:
Nice catch for those shorthands properties. But shall we always add units for properties? (cc @alejandronanez)

Reply:
I don’t know. It’s up to you. I never use units for 0 in css.

Reply:
I think we should stick to unitless, that's what we've been doing in other PRs